packages/perseus-editor/src/editor.tsx

```diff
@@ -451,13 +447,7 @@ class Editor extends React.Component<Props, State> {
             .on("copy cut", this._maybeCopyWidgets)
             .on("paste", this._maybePasteWidgets);
 
-        Dependencies.getDependencies()
-            .getKaTeX()
-            .then((katex) => {
-                this.setState({
-                    katex,
-                });
-            });
+        import("katex").then((katex) => this.setState({katex}));
```

Comment:
@jeremywiebe I think this line is causing problems for the [bundle size check](https://github.com/Khan/perseus/actions/runs/4757796426/jobs/8455053957?pr=481). The way that task is configured, it doesn't work with dynamic imports.

The reason I used a dynamic import here in the first place was to try to keep the bundle size down. The alternative would be to use the `import katex from "katex"` at the top of this file (which is currently optimized out during TS compilation, since `katex` is only being used as a type). That would cause the bundle size to increase by 160kb, though.

What do you think we should do here? Since this is the editor, and not learner-facing, maybe it's okay for the bundle size to increase?

Reply:
I agree that it's fine to let the bundle size increase.  Additionally, since we're on the path to get rid of KaTeX completely, this piece of hte editor that shows KaTeX errors won't live past the end of this project, right?

Reply:
Ideally, no, though I think there are some tradeoffs we can make in that area. KaTeX emits better error messages than MathJax (it identifies the line number and column of the errors, and gives better hints about what's wrong) so when content authors miss a curly brace or whatever, the KaTeX error helps them debug. If we wanted to replace KaTeX for this purpose, MathJax wouldn't be a drop-in. We'd probably have to write our own TeX parser.

I actually think keeping KaTeX here, just for the purpose of getting good error messages about invalid TeX syntax, would be a good-enough solution for the editor, and it would cost a lot less than writing our own parser.